lib/oj_serializers/serializer.rb

```diff
@@ -189,15 +214,32 @@ def one(item, options = nil)
     # options - list of external options to pass to the sub class (available in `item.options`)
     #
     # Returns an Oj::StringWriter instance, which is encoded as raw json.
-    def many(items, options = nil)
+    def many_as_json(items, options = nil)
       writer = new_json_writer
       write_many(writer, items, options)
       writer
     end
 
-    # Public: Creates an alias for the internal object.
-    def object_as(name)
-      define_method(name) { @object }
+    # Public: Renders the configured attributes for the specified object,
+    # without serializing to JSON.
+    #
+    # item - the item to serialize
+    # options - list of external options to pass to the sub class (available in `item.options`)
+    #
+    # Returns a Hash, with the attributes specified in the serializer.
+    def one_as_hash(item, options = nil)
```

Comment:
@ElMassimo Can you make `option` default to `{}`? Our serializer specs were raising `undefined method [] for nil` errors since `@object.options` was nil.

Reply:
Sounds unusual, as [`options` would never return `nil`](https://github.com/ElMassimo/oj_serializers/blob/main/lib/oj_serializers/serializer.rb#L38), because it falls back to `{}`.

If you provide a small reproduction I can take a look.

Reply:
Ok, did not realize there was an `options` method directly in the serializer. Our serializer is calling `@object.options` directly, which is where the `nil` error comes from.